cfgmgr/vlanmgr.cpp

```diff
@@ -579,6 +612,13 @@ void VlanMgr::doVlanMemberTask(Consumer &consumer)
         /* Other than the case of member port/lag is not ready, no retry will be performed */
         it = consumer.m_toSync.erase(it);
     }
+    if (!replayDone && m_vlanMemberReplay.empty() &&
+        WarmStart::isWarmStart())
+    {
+        replayDone = true;
+        WarmStart::setWarmStartState("vlanmgrd", WarmStart::RECONCILED);
```

Comment:
Why this change from REPLAYED?

Reply:
@prsunny test_VlanMgrdWarmRestart was failing as it expected the state to be RECONCILED. I understand now that as per HLD it is supposed to be REPLAYED. I will discuss with team members and check on how to handle this.

Reply:
@prsunny I will change the existing pytest script to match the EVPN HLD

Reply:
@prsunny I will change the existing pytest to expect REPLAYED instead of RECONCILED to match the EVPN HLD